twitter/Handler/APIManager.m

```diff
@@ -50,20 +50,54 @@ - (instancetype)init {
     return self;
 }
 
-- (void)getHomeTimelineWithCompletion:(void(^)(NSArray *tweets,
-                                               NSError *error))completion {
-    NSDictionary *parameters = @{@"tweet_mode":@"extended"};
+- (void)getHomeTimelineWithCompletion:(void(^)(NSArray<Tweet *> *tweets, NSError *error))completion {
+    NSDictionary *parameters = @{@"tweet_mode":@"extended", @"exclude_replies":@"true"};
     
     [self GET:@"1.1/statuses/home_timeline.json"
        parameters:parameters progress:nil success:^(NSURLSessionDataTask * _Nonnull task,
                                                     NSArray *  _Nullable tweetDictionaries) {
-           NSMutableArray *tweets = [Tweet tweetsWithArray:tweetDictionaries];
+           NSMutableArray<Tweet *> *tweets = [Tweet tweetsWithArray:tweetDictionaries];
            completion(tweets, nil);
        } failure:^(NSURLSessionDataTask * _Nullable task, NSError * _Nonnull error) {
            completion(nil, error);
     }];
 }
 
+- (void)getMoreHomeTimelineTweets:(NSString *)tweetId completion:(void(^)(NSArray<Tweet *> *tweets, NSError *error))completion {
```

Comment:
Interesting way to do this. I'll be the API supports some kind of paging as well. Curious about your thinking on this.